packages/affiliates/libs/uniswap/models.ts

```diff
@@ -40,7 +39,7 @@ export function Table<T extends MaybeId>(config: { makeId: MakeId<T>; type: Id }
     const got = await get(id);
     return set({ ...got, ...data });
   }
-  function entries() {
+  async function entries() {
```

Comment:
these functions are async to mimic the interface if the data was being queried from a database or remote source.